src/engraving/libmscore/spanner.cpp

```diff
@@ -1236,6 +1236,22 @@ void Spanner::setTicks(const Fraction& f)
     }
 }
 
+void Spanner::setSpecificVoiceIfNeeded(ChordRest* startCR)
+{
+    static const std::set<ElementType> voiceSpecificSpanners {
```

Comment:
std::unordered_set would be more optimal here. Also, the name should be VOICE_SPECIFIC_SPANNERS (because it is a static constant)